.github/workflows/cifuzz.yml

```diff
@@ -1,5 +1,14 @@
 name: CIFuzz
-on: [pull_request]
+on: 
+  push:
+    branches:
+      - master
+    tags:
+    paths-ignore:
+      - '**.md'
```

Comment:
Adding this one?
- 'docs/**'
- 'hack/**'
- 'CHANGELOG/**'

Reply:
files need ignoring include the below files.
docs/**(include some images) **.md OWNERS MAINTAINERS
I think this is all.

Reply:
> 'hack/**'

There are many scripts under hack dir, they need to run the ci check. :)

Reply:
There are many scripts under hack, they need to be checked. :)

Reply:
yes, hack dir should be checked